.github/workflows/publish_pypi.yml

```diff
@@ -116,6 +117,8 @@ jobs:
       - name: Build wheels on Linux
         run: pipx run cibuildwheel --output-dir wheelhouse
         env:
+          CIBW_ENVIRONMENT: >
+            PYBAMM_DISABLE_TELEMETRY="true"
```

Comment:
As far as I can tell the environment variables were not making it into the CIBW environment. This disables telemetry in the wheel tests